data/json/recipes/recipe_others.json

```diff
@@ -3391,19 +3391,21 @@
     "time": "10 m",
     "autolearn": true,
     "tools": [ [ [ "surface_heat", 4, "LIST" ] ], [ [ "mold_plastic", -1 ] ] ],
-    "components": [ [ [ "bag_plastic", 10 ] ] ]
-  },
-  {
-    "type": "recipe",
-    "result": "plastic_chunk",
-    "id_suffix": "from_cash_cards",
```

Comment:
You need to add this to recipes/recipe_obsolete.json or this will cause load errors:
```
{
    "type": "recipe",
    "result": "plastic_chunk",
    "id_suffix": "from_cash_cards",
    "obsolete": true
  }
```